shasta_install_utility_common/products.py

```diff
@@ -628,3 +850,29 @@ def uninstall_hosted_repos(self, nexus_api):
             raise ProductInstallException(
                 f'One or more errors occurred uninstalling repositories for {self.name} {self.version}.'
             )
+
+    def get_helm_chart_nexus_component_ids(self) -> Optional[NexusHelmChartTuples]:
+        """
+        Maps the names of the helm charts found in the configmap which contain (name, version).
+        Nexus requires that the component_id for API calls. This method searches the entire charts
```

Comment:
I don't understand the comment: "Nexus requires that the component_id for API calls". Is a word missing?

Reply:
delete API calls will fix